src/util/data.h

```diff
@@ -7,3 +7,5 @@
 
 #define min(a, b) (((a) < (b)) ? (a) : (b))
 #define max(a, b) (((a) > (b)) ? (a) : (b))
+
+#define REG_SET(register, mask, value) (register &= ~(register & mask) | value)
```

Comment:
register is a keyword in C, doesn't do anything here but maybe best to use something else?

Reply:
maybe I'm reading this wrong, but i think this should be `register = (register & ~mask) | value`

Reply:
Yup :sweat_smile: